components/motor/dimensionengineering/sabertooth.go

```diff
@@ -353,12 +353,12 @@ func (m *Motor) SetPower(ctx context.Context, powerPct float64, extra map[string
 	m.currentPowerPct = powerPct
 
 	rawSpeed := powerPct * maxSpeed
-	switch speed := math.Abs(rawSpeed); {
-	case speed < 0.1:
-		m.c.logger.CWarn(ctx, "motor speed is nearly 0 rev_per_min")
-	case m.maxRPM > 0 && speed > m.maxRPM-0.1:
-		m.c.logger.CWarnf(ctx, "motor speed is nearly the max rev_per_min (%f)", m.maxRPM)
-	default:
+	warning, err := motor.CheckSpeed(rawSpeed, m.maxRPM)
+	if warning != "" {
+		m.logger.CWarn(ctx, warning)
+	}
+	if err != nil {
+		return err
```

Comment:
I don't feel like this is really necessary because it will just congest the logs. when this error is returned, there will already be a warning log that says `"motor speed requested is 0 rev_per_min"` and then an error log that says `"Cannot move motor at an RPM that is nearly 0"`